playbooks/openshift-node/private/clean_image.yml

```diff
@@ -27,3 +27,41 @@
     file:
       state: absent
       path: "/var/lib/cloud/"
+
+- name: Set gquota for slash filesystem
+  hosts: oo_nodes_to_config
+  tasks:
```

Comment:
Stick all this stuff into openshift_aws, using import_role + tasks_from.  Then you can define defaults in openshift_aws/defaults/main.yml for things that are changeable via inventory.

Reply:
These tasks should run prior to cleaning up ansible_facts (previous play)